release/kdm/rke2_charts.go

```diff
@@ -0,0 +1,103 @@
+package kdm
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+type (
+	ChartsFile struct {
+		Charts []Chart `yaml:"charts"`
+	}
+
+	Chart struct {
+		Repo     string `yaml:"repo,omitempty"`
+		Version  string `yaml:"version,omitempty"`
+		Filename string `yaml:"filename,omitempty"`
+	}
+)
+
+const (
+	chartsURLFormat = "https://raw.githubusercontent.com/rancher/rke2/%s/charts/chart_versions.yaml"
+)
+
+func getChartsFromVersion(version string) (map[string]Chart, error) {
+	chartsURL := fmt.Sprintf(chartsURLFormat, version)
```

Comment:
This is only used once so we don't need the constant and printf. Concatenate the 2 here is fine.